### ISSUE TITLE
helloworld tutorial gtk TypeError: <lambda>() missing 1 required positional argument: 'data

### ISSUE THREAD

**Issue by elelay** [beeware/toga#861]
## Expected Behavior

When pressing Ctrl-Q the helloworld tutorial app exits

## Current Behavior

```
(VENV) gpodder-toga> python -m helloworld
[GTK+] Not implemented: Widget.set_hidden()
[GTK+] Not implemented: Widget.set_hidden()
[GTK+] Not implemented: Button.set_background_color()
hello
Traceback (most recent call last):
  File "/home/elelay/devel/gpodder-toga/VENV/lib/python3.8/site-packages/toga_gtk/app.py", line 21, in _handler
    cmd.action(cmd)
  File "/home/elelay/devel/gpodder-toga/VENV/lib/python3.8/site-packages/toga/handlers.py", line 55, in _handler
    result = handler(interface, *args, **kwargs)
TypeError: <lambda>() missing 1 required positional argument: 'data'
Traceback (most recent call last):
  File "/home/elelay/devel/gpodder-toga/VENV/lib/python3.8/site-packages/toga_gtk/app.py", line 21, in _handler
    cmd.action(cmd)
  File "/home/elelay/devel/gpodder-toga/VENV/lib/python3.8/site-packages/toga/handlers.py", line 55, in _handler
    result = handler(interface, *args, **kwargs)
TypeError: <lambda>() missing 1 required positional argument: 'data'
```

## Steps to reproduce

1. follow the first toga tutorial.

2. run with `python -m helloworld`. The application launches.

3. Press Ctrl-Q triggers an exception (see above)

## Your Environment

  - [ ] macOS - version:
  - [x] Linux - ArchLinux: python3.8, gtk+ 3.24
  - [ ] Windows - version:
  - [ ] Other - name:  - version:

```
(VENV) gpodder-toga> pip freeze
gbulb==0.6.1
pycairo==1.19.1
PyGObject==3.36.0
toga==0.3.0.dev19
toga-core==0.3.0.dev19
toga-gtk==0.3.0.dev19
travertino==0.1.2
```

* Toga Target (the type of app you are trying to generate)

  - [ ] android
  - [ ] cocoa
  - [ ] django
  - [x] gtk
  - [ ] iOS
  - [ ] tvOS
  - [ ] watchOS
  - [ ] winforms
  - [ ] win32
  - [ ] Other (please specify)

## Solution

This is trivially fixed by passing on the `data` parameter in app.py's [gtk_menu_activate](https://github.com/beeware/toga/blob/master/src/gtk/toga_gtk/app.py#L21) but I don't know if it applies to other platforms...

Patching `app.py` as described allows me to quit using **Ctrl-Q**

**Comment by eliotb** [comment]
I think the right change is to remove the data parameter from the lambda around line 80, rather than adding it to the call site.

**Comment by freakboy3742** [comment]
Fixed by #873.

**Comment by elelay** [comment]
yes, much better, thanks!
